### ISSUE TITLE
Any interest in appveyor wheels? 

### ISSUE THREAD

**Issue by thequackdaddy** [statsmodels/statsmodels#3786]
We live in a Windows world, and building statsmodels on windows can be a little confusing, especially because the compiler is python version-specific. Additionally, we don't have wheels on pypi for windows and statsmodels 0.8.0. (Kind of some self-interest here because at work my personal desktop I can install the windows build tools. On some other machines, I can't but I can install wheels. I love statsmodels, but lacking the ability to `pip install statsmodels` always makes me do some extra steps.)

I played with having appveyor build wheels and it seems to add about 1 minute per build... so a total of 3 minutes for the current setup. That would get us windows wheels for python 2.7 and 3.6 (most of the time... when the whole statespace bug doesn't crash). 

Appveyor  could also theoretically upload the wheels to pypi only when master is compiled (and/or on a tag).

**Comment by josef-pkt** [comment]
If you or someone can set this up, then this would be great.

**Comment by josef-pkt** [comment]
Maybe we can reuse the cloud storage that other related packages are using.
@matthew-brett  ?

**Comment by matthew-brett** [comment]
Yes, you're certainly welcome to use the Rackspace stuff for wheels.  The killer problem is that we don't have Scipy wheels for Windows...

**Comment by waynenilsen** [comment]
It looks like [scipy wheels are incoming](https://github.com/scipy/scipy/issues/5461#issuecomment-326744515). Not sure when they will land on pypi

**Comment by bashtage** [comment]
@josef-pkt Should be deferred until after 0.9.

**Comment by waynenilsen** [comment]
Scipy windows wheels have [officially landed in 1.0](https://mail.python.org/pipermail/scipy-user/2017-October/037357.html)

> Highlights of this release
> --------------------------
> 
> Some of the highlights of this release are:
> 
> - Major build improvements.  Windows wheels are available on PyPI for the
>   first time, and continuous integration has been set up on Windows and OS X
>   in addition to Linux.

**Comment by jbrockmendel** [comment]
Is this still an issue?

**Comment by matthew-brett** [comment]
Er - no, in that we now have the https://github.com/MacPython/statsmodels-wheels repo, that includes an Appveyor wheel build.  Yes, a little bit, in that that repo still hasn't generated wheels that pass the tests:

https://ci.appveyor.com/project/matthew-brett/statsmodels-wheels/build/1.0.33

Let's close this one when that does work...

**Comment by bashtage** [comment]
Master should be passing now. Is it worth checking before a release is nearly ready?

**Comment by matthew-brett** [comment]
Sadly master is not passing at the moment on the wheel builds - see https://github.com/statsmodels/statsmodels/issues/5108

**Comment by matthew-brett** [comment]
OK - it looks like master is passing now : https://travis-ci.org/MacPython/statsmodels-wheels/builds/424746178

So - I think this can be closed.
